charts/govuk-sli-collector/values.yaml

```diff
@@ -0,0 +1,40 @@
+# Default values for govuk-sli-collector.
+
+govukEnvironment: test
+
+enabled: false
+intervalMinutes: "10"
+offsetMinutes: "10"
```

Comment:
The script uses these internally, but if you'd prefer the duplication (`schedule` and `intervalMinutes`), I can do that and add an inline comment ("these should always match...")?

Reply:
Oh I see — sorry yeah in that case what you've done there makes perfect sense. Better for it to be impossible to misconfigure rather than rely on people to keep two/three things consistent.